net/ProfileViewController.m

```diff
@@ -145,7 +160,13 @@ - (void) revealAllViews {
         [self.labelSystemRole setHidden:NO];
         [self.systemRole setHidden:NO];
         
-        if (self.userID == [CurrentItems sharedItems].user.userId) {
+        if ([[self backViewController] isKindOfClass:[IssueHistoryViewController class]]) {
```

Comment:
Some sing wrong with class names. You check if backViewController is a IssueHistoryViewController, and show current user info. It should be something like CurrentUserViewController. Isn't it?